docs/python/tutorial-django-push-to-registry.md

```diff
@@ -0,0 +1,62 @@
+---
+Order: 12
+Area: python
+TOCTitle: Push Django Images to a Registry
+ContentId: 4e45a3f6-b72d-4647-82a5-22f7ee593d47
+PageTitle: Push Django Images to a Registry in Visual Studio Code
+DateApproved: 08/12/2019
+MetaDescription: How to push a Django image to a container registry using the VS Code docker extension
+MetaSocialImage: images/tutorial/social.png
+---
+
+# Push Django Images to a Registry
+
+## Create a container registry
```

Comment:
Add a sentence between these two headings that captures the essential points for the article - the first sentence here is important for search results.

Reply:
Please remove this file from the PR. I can update it in a separate PR